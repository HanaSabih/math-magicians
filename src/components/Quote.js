import Card from 'react-bootstrap/Card';

const Quote = () => (
  <Card className="w-75 border-0 mx-auto mt-5">
    <Card.Body>
      <blockquote className="blockquote mb-0">
        <p className="fs-2 mt-5">
          {' '}
          Mathematics is not about numbers, equations, computations
          or algorithms;
          <br />
          {' '}
          it&lsquo;s about understanding.
          {' '}
        </p>
        <footer className="blockquote-footer">
          <cite title="Source Title">William Paul Thurston</cite>
        </footer>
      </blockquote>
    </Card.Body>
  </Card>
);

export default Quote;
