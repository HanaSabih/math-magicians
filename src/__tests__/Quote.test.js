import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

it('renders correctly', () => {
  const testQuote = renderer.create(<Quote />).toJSON();
  expect(testQuote).toMatchSnapshot();
});
