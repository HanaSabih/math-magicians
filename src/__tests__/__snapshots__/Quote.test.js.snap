// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="w-75 border-0 mx-auto mt-5 card"
>
  <div
    className="card-body"
  >
    <blockquote
      className="blockquote mb-0"
    >
      <p
        className="fs-2 mt-5"
      >
         
        Mathematics is not about numbers, equations, computations or algorithms;
        <br />
         
        itâ€˜s about understanding.
         
      </p>
      <footer
        className="blockquote-footer"
      >
        <cite
          title="Source Title"
        >
          William Paul Thurston
        </cite>
      </footer>
    </blockquote>
  </div>
</div>
`;
